src/Microsoft.ApplicationInsights.AspNetCore/Logging/Implementation/ApplicationInsightsLoggerProvider.cs

```diff
@@ -7,7 +7,10 @@
     /// <summary>
     /// <see cref="ILoggerProvider"/> implementation that creates returns instances of <see cref="ApplicationInsightsLogger"/>
     /// </summary>
+#if !NETSTANDARD2_0
```

Comment:
For clarity sake, put the whole class in #IF?

Reply:
We cannot remove this whole class - as that'd be a breaking change for AI customers, and we dont want that. (except when we change major from to 3.XX)..